website/views/shared/avatar/index.jade

```diff
@@ -3,7 +3,7 @@ include appearance
 mixin herobox(opts)
   - if (!opts) {opts = {minimal:false,main:false}}
 
-  - var ngClick = 'spell ? castEnd(profile, "user", $event) : clickMember(profile._id)'
+  - var ngClick = 'spell ? castEnd(profile, "user", $event) : siHovered ? null : clickMember(profile._id)'
```

Comment:
Once you get a double ternary in here, we should probably extract this to a function.